posts/services/common.py

```diff
@@ -116,6 +124,41 @@ def create_post(
     return obj
 
 
+def trigger_update_post_translations(
+    post: Post, with_comments: bool = False, force: bool = False
+):
+    is_private = post.default_project.default_permission is None
+    if not force and is_private:
+        return
+
+    post.trigger_translation_if_dirty()
+    if post.question is not None:
```

Comment:
I suggest `post.question_id` checks to minimize db queries

Reply:
Good point, will change

Reply:
Done